Clean Swift/Scene.xctemplate/UIViewController/___FILEBASENAME___ViewController.swift

```diff
@@ -56,12 +56,28 @@ class ___VARIABLE_sceneName___ViewController: UIViewController, ___VARIABLE_scen
         setupFetchFromDataStore()
     }
 
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-        trackAnalytics()
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        trackScreenViewAnalytics()
+        registerNotifications()
     }
 
-    // MARK: Use Case - Fetch Data From DataStore
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        unregisterNotifications()
+    }
+
+    // MARK: - Notifications
+
+    func registerNotifications() {
+        NotificationCenter.default.addObserver(self, selector: #selector(trackScreenViewAnalytics), name: UIApplication.didBecomeActiveNotification, object: nil)
```

Comment:
```suggestion
        let selector = #selector(trackScreenViewAnalytics)
        let notification = UIApplication.didBecomeActiveNotification
        NotificationCenter.default.addObserver(self, selector: selector, name: notification, object: nil)
```

@SentulAsia Suggest to also split the parameters into separate variables, to auto comply with our line length limit that will be implemented later through the linter, later.

Reply:
Will reflect in next PR